docs/design/HealthEndpoint.md

```diff
@@ -0,0 +1,289 @@
+# Design Document: DAB Health Endpoint
+## Objective:
```

Comment:
PR description should remove the template description and provide details specific to this PR.